src/libraries/Microsoft.Extensions.Logging.Abstractions/tests/Microsoft.Extensions.Logging.Generators.Tests/TestClasses/MiscTestExtensions.cs

```diff
@@ -3,6 +3,61 @@
 
 using Microsoft.Extensions.Logging;
 
+// No explicit tests use the following two types, but the fact
+// that they are here means we exercise a constraint that we
+// exclude private fields of base types.
+// If that logic ever changes, then just by having these two classes
+// will mean that compilation fails with:
+// error SYSLIB1020: Found multiple fields of type Microsoft.Extensions.Logging.ILogger in class DerivedClass_with_private_logger
+public class BaseClass_with_private_logger
```

Comment:
nit: change to PascalCase rather than using underscore in the name. (`BaseClassWithPrivateLogger`, `DerivedClassWithPrivateLogger`, etc.)